code/modules/reagents/reagent_containers/glass/bottle.dm

```diff
@@ -250,4 +250,16 @@
 	icon_state = "rbottle"
 	New()
 		..()
-		reagents.add_reagent("formaldehyde", 50)
\ No newline at end of file
+		reagents.add_reagent("formaldehyde", 50)
+
+
+////////////////////Traitor Poison Bottle//////////////////////////////
+
+/obj/item/weapon/reagent_containers/glass/bottle/traitor
+	desc = "It has a small skull and crossbones on it. Uh-oh!"
+	possible_transfer_amounts = list(5,10,15,25,30,40)
+	volume = 40
+
+	New()
+		..()
+		reagents.add_reagent(pick("polonium","initropidril","concentrated_initro","pancuronium","sodium_thiopental","ketamine","sulfonal","amanitin","coniine","curare","sarin","histamine","venom","cyanide"), 40)
```

Comment:
Dont put anything I can already mix in here, it's a waste of 2 tc if I end up with cyanide when I could of just made it